src/database/mysql.test.ts

```diff
@@ -0,0 +1,155 @@
+import { describe, it, expect, beforeEach, afterEach } from "vitest";
```

Comment:
I didn't aware `vitest` need to import so many things 😱 It seems more convenient to switch to `jest`. What do you think?